interface/patient_file/history/history.php

```diff
@@ -18,20 +18,16 @@
 ?>
 <html>
 <head>
-<?php html_header_show();?>
```

Comment:
I wouldn't remove the above, since it seems that some folks use this as a hook for custom stuff.